src/medicalpage/MedicalPage.js

```diff
@@ -1,137 +1,31 @@
-import React, { useState } from 'react';
+import React from 'react';
 import Header from '../components/Header';
 import ContentContainer from '../components/ContentContainer';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { library } from '@fortawesome/fontawesome-svg-core';
-import {
-  faClinicMedical,
-  faPrescriptionBottle,
-  faCommentMedical,
-  faCheckSquare,
-  faEdit
-} from '@fortawesome/free-solid-svg-icons';
-import styled from 'styled-components';
-import Label from '../components/Label';
-import Input from '../components/Input';
-import MedicalIcon from './MedicalIcon';
-
-library.add(
-  faClinicMedical,
-  faPrescriptionBottle,
-  faCommentMedical,
-  faCheckSquare,
-  faEdit
-);
-
-const LocationInformationContainer = styled.section`
-  display: grid;
-  grid-template-rows: auto auto auto;
-  grid-template-columns: auto auto auto auto;
-  grid-gap: 10px;
-  padding: 10px;
-  border: grey solid 1px;
-  border-radius: 10px;
-  margin-bottom: 20px;
-  align-items: center;
-`;
-
-const EditIcon = styled(FontAwesomeIcon)`
-  color: skyblue;
-  font-size: 1.5rem;
-`;
-
-const Output = styled.div``;
-
-export default function MedicalPage({ location, onLocationChange }) {
-  const { adress, phonenum, room } = location;
-  const [isAdressEditable, setIsAdressEditable] = useState(false);
-  const [isPhonenumEditable, setIsPhonenumEditable] = useState(false);
-  const [isRoomEditable, setIsRoomEditable] = useState(false);
-
-  function onAdressChange(name, event, onLocationChange) {
-    const newLocation = { ...location, [name]: event.target.value };
-    onLocationChange(newLocation);
-  }
-
+import LocationInformation from './LocationInformation';
+import MedicalInformation from './MedicalInformation';
+
+export default function MedicalPage({
+  location,
+  onLocationChange,
+  medicationList,
+  onSingleMedicationSubmit,
+  onSingleMedicationDelete,
+  onSingleMedicationChange
+}) {
   return (
     <>
       <Header>Informationen</Header>
```

Comment:
You could render the `<Header>` dynamically from the `App.js` page (with the page title as `props`). This would probably affect your grid styling though, so **beware**!